packages/errors/src/messages.ts

```diff
@@ -196,7 +196,12 @@ export const SolanaErrorMessages: Readonly<{
     // TypeScript will fail to build this project if add an error code without a message.
     [P in SolanaErrorCode]: string;
 }> = {
-    [SOLANA_ERROR__ACCOUNT_NOT_FOUND]: 'Account not found at address: $address',
+    [SOLANA_ERROR__ACCOUNTS_ACCOUNT_NOT_FOUND]: 'Account not found at address: $address',
+    [SOLANA_ERROR__ACCOUNTS_EXPECTED_ALL_ACCOUNTS_TO_BE_DECODED]:
+        'Not all accounts were decoded. Encoded accounts found at addresses: $addresses.',
```

Comment:
But what's your opinion of error messages that end in periods? :)

Reply:
I don't mind either way but it should be all or nothing IMO. 🫡